server/publications/collections/products.js

```diff
@@ -147,6 +147,10 @@ function filterProducts(productFilters) {
       }, {
         slug: { $in: shopIdsOrSlugs }
       }]
+    }, {
+      fields: {
+        "_id": 1
+      }
```

Comment:
not required for this bug fix, but since we only care about the _id, we should save some data transfer by only requesting _id from the db